src/components/Gradebook/StatusAlerts.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { StatusAlert } from '@edx/paragon';
+
+import { closeBanner } from '../../data/actions/grades';
+
+export class StatusAlerts extends React.Component {
+  getCourseGradeFilterAlertDialog() {
```

Comment:
I don't like using Dialog here as that is an overloaded term that generally means something akin to a modal in this space.
Also, this could be a getter, as that's how you're using it :-)
option: `get courseGradeFilterAlertText` or `get courseGradeFilterAlertDialogText`